packages/sdk/src/conversation/sso/defaultSsoExecutionActivityHandler.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  BotState,
+  ConversationState,
+  MemoryStorage,
+  SigninStateVerificationQuery,
+  StatePropertyAccessor,
+  TeamsActivityHandler,
+  TurnContext,
+  UserState,
+} from "botbuilder";
+import { TeamsBotSsoPromptSettings } from "../../bot/teamsBotSsoPrompt";
+import { TeamsFx } from "../../core/teamsfx";
+import { IdentityType } from "../../models/identityType";
+import { SsoConfig, SsoExecutionActivityHandler, TeamsFxBotSsoCommandHandler } from "../interface";
+import { SsoExecutionDialog } from "./ssoExecutionDialog";
+
+/**
+ * Default sso execution activity handler
+ */
+export class DefaultSsoExecutionActivityHandler
+  extends TeamsActivityHandler
+  implements SsoExecutionActivityHandler
+{
+  private ssoExecutionDialog: SsoExecutionDialog;
+  private userState: BotState;
+  private conversationState: BotState;
+  private dialogState: StatePropertyAccessor;
+
+  /**
+   * Creates a new instance of the DefaultSsoExecutionActivityHandler.
+   * @param ssoConfig configuration for sso command bot
+   */
+  constructor(ssoConfig: SsoConfig | undefined) {
+    super();
+    const memoryStorage = new MemoryStorage();
+    const userState = ssoConfig?.userState ?? new UserState(memoryStorage);
+    const conversationState = ssoConfig?.conversationState ?? new ConversationState(memoryStorage);
+    const dedupStorage = ssoConfig?.dedupStorage ?? memoryStorage;
+    const scopes = ssoConfig?.scopes ?? ["User.Read"];
+
+    const teamsfx = new TeamsFx(IdentityType.User, ssoConfig?.teamsFxConfig);
+    const settings: TeamsBotSsoPromptSettings = {
+      scopes: scopes,
+      timeout: ssoConfig?.ssoPromptConfig?.timeout,
+      endOnInvalidMessage: ssoConfig?.ssoPromptConfig?.endOnInvalidMessage === false ? false : true,
```

Comment:
`endOnInvalidMessage` is also optional, may be able to assign `ssoConfig?.ssoPromptConfig?.endOnInvalidMessage` to it directly

Reply:
Thanks, will update~